bin/worker.js

```diff
@@ -156,27 +158,22 @@ co(function *() {
   // Default to always having at least a capacity of one.
   config.capacity = config.capacity || 1;
 
-  var statsdConf = url.parse(workerConf.get('statsd:url'));
-  // Raw statsd interface.
-  config.statsd = new SDC({
-    debug: !!process.env.DEBUG,
-    // TOOD: Add real configuration options for this.
-    host: statsdConf.hostname,
-    port: statsdConf.port,
-    // docker-worker.<worker-type>.<provisionerId>.
-    prefix: sanitizeGraphPath(
-      config.statsd.prefix,
-      'docker-worker',
-      config.provisionerId || 'unknown',
-      config.workerGroup || 'unknown',
-      config.workerType || 'unknown',
-      config.workerNodeType || 'unknown'
-    )
-  });
-
   // Wrapped stats helper to support generators, etc...
-  config.stats = new Stats(config.statsd);
-  config.stats.increment('started');
+  config.stats = new Stats(config);
+  config.stats.record('workerStart', Date.now()-os.uptime() * 1000);
+
+
+  // Only catch these metrics when running on aws host.  Running within
+  // test environment causes numerous issues.
+  if (host === 'aws') {
+    base.stats.startProcessUsageReporting({
+      drain: config.stats.influx,
+      component: 'docker-worker',
+      process: 'docker-worker'
```

Comment:
Hmm... we should modify this in `base` to take `tags` like the thing I suggested for in the PR about client API stats...